src/Makefile

```diff
@@ -33,7 +33,7 @@ export GO111MODULE=on
 $(BIN):
 	@mkdir -p $@
 $(BIN)/%: | $(BIN) ; $(info $(M) building $(PACKAGE)…)
-	$Q tmp=$$(mktemp -d); \
+	   tmp=$$(mktemp -d); \
```

Comment:
Is this wanted?

Reply:
I think this was added during debugging - removed now.